src/index.jsx

```diff
@@ -34,12 +34,36 @@ const apiClient = getAuthenticatedAPIClient({
   loggingService: NewRelicLoggingService,
 });
 
+let callbackAccessToken = null;
```

Comment:
What are the issues with using a global variable here?

Reply:
I’m not clear. Do you think there is an issue? It is scoped to the module.